<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionEditWidget</class>
 <widget class="QWidget" name="VersionEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VersionEditWidger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="VersionLineEdit" name="major_version"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VersionLineEdit" name="minor_version"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VersionLineEdit" name="release_version"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VersionLineEdit</class>
   <extends>QLineEdit</extends>
   <header>versionlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
